modules/v2x/proto/v2x_traffic_light_detection.proto

```diff
@@ -0,0 +1,58 @@
+syntax = "proto2";
+
+package apollo.v2x;
+
+import "modules/common/proto/header.proto";
+import "modules/perception/proto/traffic_light_detection.proto";
+
+message TrafficLightBox{
```

Comment:
Why not use TrafficLightBox defined in modules/perception/proto/traffic_light_detection.proto?

Reply:
changed in #12368